syncthing/manifest.json

```diff
@@ -50,12 +50,12 @@
       "Syncthing Overview": "assets/dashboards/syncthing_overview.json"
     },
     "monitors": {
-      "[Syncthing] Disconnected": "assets/monitors/syncthing_disconnected.json",
-      "[Syncthing] Out of sync": "assets/monitors/syncthing_out_of_sync.json",
-      "[Syncthing] System error": "assets/monitors/syncthing_system_error.json",
-      "[Syncthing] Folder error": "assets/monitors/syncthing_folder_error.json",
-      "[Syncthing] Service error": "assets/monitors/syncthing_service_error.json",
-      "[Syncthing] Device not connected": "assets/monitors/syncthing_device_not_connected.json"
+      "No active connections": "assets/monitors/syncthing_disconnected.json",
+      "Files out of sync": "assets/monitors/syncthing_out_of_sync.json",
+      "System errorsr": "assets/monitors/syncthing_system_error.json",
+      "Folder errors": "assets/monitors/syncthing_folder_error.json",
+      "Service is failed": "assets/monitors/syncthing_service_error.json",
```

Comment:
```suggestion
      "Service failed": "assets/monitors/syncthing_service_error.json",
```

Reply:
I'll have to leave this as is to match the title in `syncthing/assets/monitors/syncthing_service_error.json`